### ISSUE TITLE
Adding a new password fails with .gpg-id found but invalid key id

### ISSUE THREAD

**Issue by Systemk2** [zeapo/Android-Password-Store#1277]
When trying to add a password using the "New password" dialog, a message appears which in German is "".gpg-id gefunden, enthält aber eine ungültige Schlüssel-ID, Fingerabdruck oder Benutzer-ID" (= "found .gpg-id, but contains an invalid key ID, fingerprint or user ID")

1. Open Pasword Store
2. Press "+"
3. Add name and password

**Expected behaviour**
Password is stored

**Device information (please complete the following information):**
 - Device: Motorola moto G(7) Power
 - OS: Android 10
 - App version 1.13.2

**Additional context**
* Decryption of passwords works without problem
* Not sure if that is related, but I made a mistake during setup: I selected the pgp key instead of the ssh key at first, so initial git clone failed, but corrected that later and everything worked (apart from password file creation)

**Comment by msfjarvis** [comment]
If you can share your key ID format here then it'd be helpful to diagnose the exact problem, otherwise we have a mechanism in place to fix it for you.

Enable hidden files and folders, then go back and delete the `.gpg-id` file. The next time you create a password, the app will ask you to select a key from OpenKeychain and will recreate the `.gpg-id` file with a more ubiquitous format that is understood by both GnuPG and OpenKeychain.

**Comment by Systemk2** [comment]
Thanks a lot for the quick reply! The workaround did it.

I deleted ~/password-store/.gpg-id on my PC, pushed the change to git, updated in Android Password Store and was there prompted to select a key. I did that and it works. 

The newly generated .git-id looks different from the old one:

Old, not working .gpg-id:

`$ cat ~/.password-store/.gpg-id`
`Passwordstore`

Newly created working gpg-id:

`$ cat .gpg-id `
`0x4cd8caed4cec028d`

Additional information: On my PC, I initially created a GPG Key pair with the user ID "Passwordstore" and initialized  pass with the command
`pass init "Passwordstore"`

So basically, it seems that the problem is that on my PC, the user ID ("Passwordstore") was set in .gpg-id, but Password Store apparently expected a Key ID  ("0x4cd8caed4cec028d"), not the User ID

**Comment by msfjarvis** [comment]
Ah I see. That's the bug from #998, that's what the workaround is for.

**Comment by Systemk2** [comment]
As this is a known bug and the workaround works fine , I close this issue. Thanks again for the perfect support.

**Comment by msfjarvis** [comment]
Re-opening since I wanna address this in documentation as well.
